### PR TITLE
Resolves #2052

### LINKED ISSUE #2052: Remove-ADUser -Confirm parameter notes
I think it's kind of silly that the very important notes about the -Confirm parameter for Remove-ADUser are listed in the "Notes" section at the bottom of the page and not in the parameter's own section.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 216a4b07-b99d-9b0e-2784-f0a501e83c3e
* Version Independent ID: b692b387-8334-3ad7-4144-eb9daafe1040
* Content: [Remove-ADUser (addsadministration)](https://docs.microsoft.com/en-us/powershell/module/addsadministration/remove-aduser?view=win10-ps)
* Content Source: [docset/windows/addsadministration/Remove-ADUser.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/addsadministration/Remove-ADUser.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix note about confirm parameter
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2052

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 

@dariomws @get-itips @tseward

cc

File: docset/winserver2012-ps/activedirectory/Remove-ADUser.md [hunk 1]
```diff
@@ -22,90 +22,68 @@ Remove-ADUser [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-Credential <PSCred
...
 ```
-C:\PS>Get-ADUser "cn=glenjohn,dc=appnc" -Server Lds.Fabrikam.com:50000 | Remove-ADUser
-```
-
-Description
-
------------
 
-Get the user with DistinguishedName 'cn=glenjohn,dc=appnc' from the AD LDS instance and remove it.
+This command gets the user with the distinguished name cn=glenjohn,dc=appnc from the AD LDS instance and removes it.s
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Remove-ADUser.md:68]:
```suggestion
This command gets the user with the distinguished name cn=glenjohn,dc=appnc from the AD LDS instance and removes it.
```

File: docset/winserver2012-ps/activedirectory/Remove-ADUser.md [hunk 2]
```diff
@@ -130,29 +108,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Credential
```
Comment (dariomws) [inline docset/winserver2012-ps/activedirectory/Remove-ADUser.md:111]:
```suggestion
```

**Comment by get-itips** [comment]
agree with @dariomws suggestions, great work @velkovb one small suggestion, I see that you include some formatting changes on the _specific resolves issue_ commit, like removing some extra whitespace, can that be moved to the fix formatting commit for future PRs? Thank you

File: docset/winserver2012r2-ps/addsadministration/Remove-ADUser.md [hunk 3]
```diff
@@ -124,15 +124,15 @@ If the cmdlet is run from such a provider drive, the account associated with the
 To specify this parameter, you can type a user name, such as User1 or Domain01\User01 or you can specify a **PSCredential** object.
 If you specify a user name for this parameter, the cmdlet prompts for a password.
 
-You can also create a **PSCredential** object by using a script or by using the Get-Credentialhttp://go.microsoft.com/fwlink/?LinkID=293936 cmdlet.
-You can then set the **Credential** parameter to the **PSCredential** object.
+You can also create a **PSCredential** object by using a script or by using the **Get-Credential** cmdlet.
+You can then set the *Credential* parameter to the **PSCredential** object.
 
-If the acting credentials do not have directory-level permission to perform the task, Active Directory PowerShell returns a terminating error.
+If the acting credentials do not have directory-level permission to perform the task, Active Directory PowerShell returns a terminating error
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Remove-ADUser.md:130]:
```suggestion
If the acting credentials do not have directory-level permission to perform the task, Active Directory PowerShell returns a terminating error.
```

Comment (JohanFreelancer9) [reply docset/winserver2012r2-ps/addsadministration/Remove-ADUser.md:130]: I'm not sure why the period was removed but it should be there.

**Comment by velkovb** [comment]
> agree with @dariomws suggestions, great work @velkovb one small suggestion, I see that you include some formatting changes on the _specific resolves issue_ commit, like removing some extra whitespace, can that be moved to the fix formatting commit for future PRs? Thank you

That is VS Code trimming white space automatically on save :) I have disabled it as sometimes it causes issues with markdown..

**Comment by get-itips** [comment]
> > agree with @dariomws suggestions, great work @velkovb one small suggestion, I see that you include some formatting changes on the _specific resolves issue_ commit, like removing some extra whitespace, can that be moved to the fix formatting commit for future PRs? Thank you
> 
> That is VS Code trimming white space automatically on save :) I have disabled it as sometimes it causes issues with markdown..

Right, thank you

File: docset/windows/addsadministration/Remove-ADUser.md [hunk 4]
```diff
@@ -49,28 +49,28 @@ To specify a default naming context for an AD LDS environment, set the **msDS-de
...
-```
+```powershell
 PS C:\> Remove-ADUser -Identity "CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM"
 ```
 
 This command removes the user with the distinguished name CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM.
 
 ### Example 4: Get a user by distinguished name and remove it
-```
+```powershells
```
Comment (tseward) [inline docset/windows/addsadministration/Remove-ADUser.md:73]: `powershells` should be singular, `powershell`

File: docset/winserver2012-ps/activedirectory/Remove-ADUser.md [hunk 5]
```diff
@@ -22,90 +22,68 @@ Remove-ADUser [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-Credential <PSCred
...
------------
+This command removes the user with the distinguished name CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM.
 
-Remove the user with DistinguishedName 'CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM'.
-
-### -------------------------- EXAMPLE 4 --------------------------
-```
-C:\PS>Get-ADUser "cn=glenjohn,dc=appnc" -Server Lds.Fabrikam.com:50000 | Remove-ADUser
+### Example 4: Get a user by distinguished name and remove it
+```powershells
```
Comment (tseward) [inline docset/winserver2012-ps/activedirectory/Remove-ADUser.md:64]: `powershells` should be singular, `powershell`

File: docset/winserver2012r2-ps/addsadministration/Remove-ADUser.md [hunk 6]
```diff
@@ -48,29 +48,29 @@ To specify a default naming context for an AD LDS environment, set the **msDS-de
...
+```powershell
+PS C:\> Remove-ADUser -Identity "CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM"
 ```
 
 This command removes the user with the distinguished name CN=Glen John,OU=Finance,OU=UserAccounts,DC=FABRIKAM,DC=COM.
 
 ### Example 4: Get a user by distinguished name and remove it
-```
-PS C:\>Get-ADUser -Identity "cn=glenjohn,dc=appnc" -Server Lds.Fabrikam.com:50000 | Remove-ADUser
+```powershells
```
Comment (tseward) [inline docset/winserver2012r2-ps/addsadministration/Remove-ADUser.md:72]: `powershells` should be singular, `powershell`

**Pull request merged**